tcf_website/static/common/sortClasses.js

```diff
@@ -83,6 +102,37 @@ const sortByGpa = () => {
     selectOrderOfSort(sortGpaAsc, "#gpa-sort-btn", "GPA");
 };
 
+const sortByRecency = () => {
```

Comment:
And then add a comment here saying that this doesn't need those checks because it's only used on 1 page

Reply:
Yup, just added it!